### ISSUE TITLE
Modeling needs high level intellectual content descriptions

### ISSUE THREAD

**Issue by azaroth42** [hybox/project-planning#53]
Responsible: @hannahfrost @anarchivist 
Accountable: @tomcramer ?
Consulted: Community?
Informed: @azaroth42 @no-reply @ggeisler 

In order to get beyond very basic modeling, further requirements are needed, especially as concerns intellectual descriptions of content.  These should be specific to the content domain (archives vs email vs scholarly content vs ...) and thus depends on #51.

This should be ongoing, and start with absolute requirements and progress towards nice to haves.
